From 0590ca8508488c96b55e2faae4721107a41691ef Mon Sep 17 00:00:00 2001
From: mvzlb <edv@mvz-labor-lb.de>
Date: Thu, 15 Dec 2022 11:59:14 +0100
Subject: [PATCH] hwdb: Fix mount matrix for CSL Panther Tab HD (#25752)

Commit a76d7aca sets ACCEL_MOUNT_MATRIX to match the device's casing
(landscape) instead of the LCD panel (portrait).

(cherry picked from commit 109197df7a9b3795e46fc69c99eb7800d2799d2b)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 07b1c1625e..39abf4db19 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -288,7 +288,7 @@ sensor:modalias:acpi:KIOX000A*:dmi:*:svnConnect:pnTablet9:*
 #########################################
 # CSL Panther Tab HD
 sensor:modalias:acpi:KIOX000A*:dmi:*:svnCSL*Computer*:pnCSL*Panther*Tab*HD:*
- ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
+ ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
 
 #########################################
 # Cube
